modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,115 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-0_{context}"]
+= Release notes for {gitops-title} 1.12.0
+
+{gitops-title} 1.12.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-12.0_{context}"]
+== Errata updates
+
+[id="gitops-1-12-0-security-update-advisory_{context}"]
+=== RHSA-xxxx:nnnn - {gitops-title} 1.12.0 security update advisory
+
+Issued: 2024-03-14
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2023:6220[RHSA-xxxx:nnnn]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+*The Title and errata number will change after the errata state moves from PUSH STATE to SHIPPED LIVE. The current entry is simply a placeholder at the moment*.
+
+[id="new-features-1-12-0_{context}"]
+== New features
+
+* With this update, the Argo CD CLI is supported and accessible as a productized and easily-installable RPM. The Argo CD CLI tool enables users to manage {gitops-title} from a terminal. Using this tool, users can manage ArgoCD resources, such as `Applications`, `ApplicationSets`, `AppProjects`, `User accounts`, and `GPG keys` from a client terminal. link:https://issues.redhat.com/browse/GITOPS-3389[GITOPS-3389]
+
+[NOTE]
+====
+The `argocd` executable binary file is included in the archives and the RPMs.
+====
+
+[WARNING]
+==== 	
+The Argo CD CLI tool is a Technology Preview feature.
+====
+
+* With this update, users can configure notifications in a dedicated `NotificationsConfiguration` custom resource (CR). This configuration replaces the workflow in earlier releases of {gitops-title}. To configure notifications in earlier releases, you had to edit the `argocd-notifications-cm` ConfigMap manually. Users can now set `Triggers`, `Templates`, `Services`, and `Subscriptions` as fields in the new `NotificationsConfiguration` custom resource (CR), which is managed by the {gitops-title} Operator. link:https://issues.redhat.com/browse/GITOPS-4130[GITOPS-4130]
```

Comment:
users/you ? Why remove the "you" perspective?

Reply:
Any way of avoiding stacked admonitions? Maybe use the following example to build "Technology Preview" into the title?

![Screenshot from 2024-03-12 16-31-13](https://github.com/openshift/openshift-docs/assets/57954076/b3f8badb-cc0a-43ec-9333-d87f04545ca7)

![Screenshot from 2024-03-12 16-32-06](https://github.com/openshift/openshift-docs/assets/57954076/826a5bd4-35d7-4572-9c5e-70c8ab6673ab)

Reply:
Indent a note by using the plus "+" character.

Reply:
> users/you ? Why remove the "you" perspective?

Fixed. Trying to make it as a standard "you" perspective throughout the RN content.

Reply:
> Indent a note by using the plus "+" character.

Fixed.

Reply:
> Any way of avoiding stacked admonitions? Maybe use the following example to build "Technology Preview" into the title?
> 
> ![Screenshot from 2024-03-12 16-31-13](https://private-user-images.githubusercontent.com/57954076/312161484-b3f8badb-cc0a-43ec-9333-d87f04545ca7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAzMTEwMTUsIm5iZiI6MTcxMDMxMDcxNSwicGF0aCI6Ii81Nzk1NDA3Ni8zMTIxNjE0ODQtYjNmOGJhZGItY2MwYS00M2VjLTkzMzMtZDg3ZjA0NTQ1Y2E3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEzVDA2MTgzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcxNTIwODQwNmQ0ZjJhNjkzZmU5ZjE5OGM5MjkxZjliNmUzZDY5M2U5MDY2MjdkMjhhNmMyZTM0NTA3NDBkYzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.I45z-EgBVVo0tETdhDQCJP-_blsnACICUonbe8nqexM)
> 
> ![Screenshot from 2024-03-12 16-32-06](https://private-user-images.githubusercontent.com/57954076/312161487-826a5bd4-35d7-4572-9c5e-70c8ab6673ab.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAzMTEwMTUsIm5iZiI6MTcxMDMxMDcxNSwicGF0aCI6Ii81Nzk1NDA3Ni8zMTIxNjE0ODctODI2YTViZDQtMzVkNy00NTcyLTljNWUtNzBjOGFiNjY3M2FiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEzVDA2MTgzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdkMjVkOGVjZjgyODJiMDJmZTllYzlkMWQ4NjZmYzcwYmFlOTc4NTI5ZWE0YWZiNzYyNGZjMmExMTAxYTAzMWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wHjj2lwYNP-BjfyTmRoCDYmdL4XTqXMu2S2V7Hzo77M)

@dfitzmau - I am unsure about how to implement your suggestion?. Could you please elaborate?

Reply:
With this update, the Argo CD CLI is supported and accessible as a productized and is an easier method for installation because it uses an RPM binary versus XM. 
+
TECH PREVIEW warning
+
By using the Argo CD CLI tool, you can complete the following tasks:
+
Manage Red Hat OpenShift GitOps from a terminal.
Manage ArgoCD resources, such as Applications, ApplicationSets, AppProjects, User accounts, and GPG keys from a client terminal. [GITOPS-3389](https://issues.redhat.com/browse/GITOPS-3389)
+
NOTE

Reply:
How about this to avoid two notes on top of each other (stacked notes).

Reply:
Fixed.